README.md

```diff
@@ -120,3 +120,8 @@ default_janus_csp_rule =
       do: "wss://#{default_janus_host}:#{janus_port} https://#{default_janus_host}:#{janus_port} https://#{default_janus_host}:#{janus_port}/meta",
       else: ""
 ```
+
+4. Update the PosgreSQL database connection string:
```

Comment:
Can you elaborate on where this config needs to be changed? It looks like a coturn server config?
Also, minor typo here, it's "PostgreSQL", not "PosgreSQL".

Reply:
This is really a Dialog related setting but as there was a  *Run Reticulum against a local Dialog instance* section here I thought it might make sense to have it here. I can move it to the dialog Readme if that makes more sense.

Reply:
Leaving it here is fine with me, but it would be useful to describe it more in the README and add context. Mainly we should leave instructions about where this dialog setting is so that readers know where to go to change it.

Reply:
Right, I've updated to add more context.